packages/adapter-node/index.js

```diff
@@ -111,3 +114,27 @@ export default function (opts = {}) {
 		}
 	};
 }
+
+/**
+ * Load sourcemaps for files in the tmp directory so that the final ones
+ * point to the original source files, instead of the generated files in outDir.
+ *
+ * @param {string} tmp
+ * @returns {import('rollup').Plugin}
+ * */
```

Comment:
The last line looks a bit malformed. Also, I just did a quick search with `grep -B 1 -r @param packages/kit/src/` and it looks like our convention is to not have a blank line
```suggestion
 * @param {string} tmp
 * @returns {import('rollup').Plugin}
 */
```